model/pytorch/tests/test_pytorchnet.py

```diff
@@ -13,30 +14,75 @@
 
 
 class ConvNet(nn.Module):
-    def __init__(self):
+    def __init__(self, in_channels=3, num_features=3):
         super(ConvNet, self).__init__()
 
-        self.conv1 = nn.Conv2d(
-            in_channels=3, out_channels=32, kernel_size=5, padding=2
+        self.layers = []
```

Comment:
Again, all these changes in this class are just a different way of defining the layers. I don't think they are related to the issue (The function `create_layer` is not called in this case as it is already a PyTorch Module)

Reply:
Yeah its not related ,  I will revert this particular section